Cargo.toml

```diff
@@ -19,13 +19,15 @@ prost-derive = "0.4"
 serde = "1.0"
 serde_derive = "1"
 structopt = "0.2"
-navitia_model = { git = "https://github.com/CanalTP/navitia_model.git", rev = "724b1b3d77ce9871cb02ff87c2d78fe64b2b27c0" }
+navitia_model = { git = "https://github.com/antoine-de/navitia_model.git", rev = "a28c1dc19cbc1fa137c90c169a14b9285cfbbf6a" }
```

Comment:
Are we forking forever?

Reply:
nop, the fork is just temporary, for https://github.com/CanalTP/navitia_model/pull/190 to be merged